lib/tier-node.js

```diff
@@ -73,6 +73,7 @@ function handler (handle, params, req, res, body, query) {
   if (!handle) {
     res.statusCode = 404
     res.end()
+    return
```

Comment:
Is covered [here](https://github.com/mcollina/fastify/blob/95820e098249578d1367facd13551417803d71ed/test/internals/tier-node.test.js#L80-L92) :)